SPECS/cert-manager/cert-manager.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "cert-manager-1.12.12-vendor.tar.gz": "eb2c70859fb2b73880f682e0c69eaeeec523481f94386b7d0150440799d7eecc",
+  "cert-manager-1.12.12-vendor.tar.gz": "f2db06fd274857c169a2906497b9d09b9cb48846231b875b7b13c7b58173ddd2",
```

Comment:
Double checking: Is this a typo or intended ?

Reply:
Local build for the cert-manager did it.
WARN[0006][srpmpacker] Updating signature for (cert-manager-1.12.12-vendor.tar.gz) from (eb2c70859fb2b73880f682e0c69eaeeec523481f94386b7d0150440799d7eecc) to (f2db06fd274857c169a2906497b9d09b9cb48846231b875b7b13c7b58173ddd2)